source/plugins/devices/wyapp/devices/raspberrypi/index.js

```diff
@@ -76,6 +76,38 @@ export function setup (options, imports, register)
 		run (project)
 		{
 			project;
+		},
+
+		/**
+		 * Modidify the project before deploy
+		 * @param {Project} project - the project
+		 */
+		deploy (project)
+		{
+			let dockerfile = null;
+			// TODO add here the dockerfile
+			if(project.language === 'nodejs')
+			{
+				dockerfile = 'FROM balenalib/raspberrypi3-debian-node:latest';
+			}
+			else{
+				dockerfile = 'FROM balenalib/raspberrypi3-debian-python:latest';
+			}
+
+			dockerfile += '\nCOPY . .\n';
+
+			dockerfile += studio.projects.getEnvironmentSetup(project);
+
+			if(project.language === 'nodejs')
+			{
+				dockerfile += 'CMD node main.js\n';
+			}
+			else
+			{
+				dockerfile += 'CMD python3 main.py';
+			}
```

Comment:
This should be delegated to the language plugin, each device knows its commands.